### PR TITLE
Upgrade Phoenix Slides.app to 1.4.0

### PR DESCRIPTION
**Pull request by nicolasm**
Phoenix Slides was updated to 1.4 more than one year ago.
http://blyt.net/phxslides/

### CODE REVIEW

File: Casks/phoenix-slides.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask :v1 => 'phoenix-slides' do
-  version '1.3.1'
-  sha256 '6e5df9e586bb33b79249d9917f99f50395f255b85bf39c5fafca64fe66aef758'
+  version '140'
```
Comment (vitorgalvao) [inline Casks/phoenix-slides.rb]: Should be `1.4.0`.

Comment (nicolasm) [inline Casks/phoenix-slides.rb]: I know but the url is:
http://blyt.net/phxslides/phoenix-slides-140.dmg

Comment (jawshooah) [inline Casks/phoenix-slides.rb]: See that bit in the `url` stanza that says `version.delete('.')`? That takes care of removing the periods for us. Nifty little tricks like that are part of what makes Ruby a great platform for our DSL :smile:

Comment (nicolasm) [inline Casks/phoenix-slides.rb]: Oh sorry. I didn't see it! I must have had dust in my eyes. ;)
Thanks

Comment (jawshooah) [inline Casks/phoenix-slides.rb]: No problem. Squash your commits and you're good to go!

**Pull request merged**